src/stores/queueStore.ts

```diff
@@ -220,6 +220,13 @@ export class TaskItemImpl {
     }
   }
 
+  public goToNode(node: ComfyNode | string | number) {
```

Comment:
I don't think `goToNode` should be here, as this method does not even reference `this`. I would suggest move it to `app.ts` for now.